test/jdk/sun/net/www/http/HttpClient/MultiThreadTest.java

```diff
@@ -63,6 +78,15 @@ static void debug(String msg) {
 
     static final AtomicInteger reqnum = new AtomicInteger();
 
+    // The number of time an idle timeout was detected by any worker
+    // on the server side.
+    static final AtomicInteger IDLE_TIME_EXCEEDED = new AtomicInteger();
```

Comment:
IDLE_TIME_EXCEEDED is allocated and incremented, but not referenced anywhere else. Did you mean to include it in a debug log or something?

Reply:
Yes - I meant to use it but in the end it was also necessary to include the time the worker has been idle since it last read/wrote a byte and the end of the test (the time at which all requests had been sent) - so I ended up not using it.  I could remove it.